tests/end-to-end/director/scopes_test.go

```diff
@@ -0,0 +1,44 @@
+package director
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestScopesAuthorization(t *testing.T) {
+	// given
+	ctx := context.Background()
+	id := uuid.New().String()
+	requiredScopes, err := tc.scopeProvider.GetRequiredScopes("query.applicationsForRuntime")
+	require.NoError(t, err)
+
+	testCases := []struct {
+		Name                 string
+		UseDefaultScopes     bool
+		Scopes               []string
+		ExpectedErrorMessage string
+	}{
+		{Name: "Required Scopes", Scopes: requiredScopes, ExpectedErrorMessage: "runtime does not exist"},
```

Comment:
this is really strange that in the test for scopes you are expecting error: `runtime does not exist`